src/idiom128.fuzion

```diff
@@ -0,0 +1,8 @@
+BreadthFirstTraverse(f node -> unit, root node) =>
+  queue := [root].asList
+  BreadthFirstTraverse f queue
+  BreadthFirstTraverse(f node -> unit, queue List<node>) =>
```

Comment:
I would give the inner feature a different name. Currently, it shadows the outer feature (which has the same name/argcount), I might decide to create an error if a feature is shadowed like this.